doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -131,11 +131,16 @@ already supported by upstream Kubernetes. For some applications, Huge Pages
 should be allocated to account for consideration of the underlying HW topology.
 This newer feature is missing from Kubernetes, therefore a gap has been
 identified and added to [Chapter 6.2.8](./chapter06.md#628-hw-topology-aware-hugepages)
-<a name="3.2.1.2"></a>
-#### 3.2.1.2 HW Topology management
 
-> This chapter should describe considerations about hardware topology
-management.
+<a name="3.2.1.4"></a>
+#### 3.2.1.4 Hardware Topology Management
+
+Scheduling pods across NUMA boundaries can result in lower performance and higher latencies for applications that require optimizations of CPU isolation, memory and device locality.
```

Comment:
I don't think the qualification at the end is necessary
```suggestion
Scheduling pods across NUMA boundaries can result in lower performance and higher latencies for applications.
```

Reply:
> I don't think the qualification at the end is necessary

@pgoyal01 : How about keeping it in separate sentence? Please check current text.